crates/ruma-common/src/encryption.rs

```diff
@@ -156,7 +156,7 @@ impl CrossSigningKey {
 
 /// The usage of a cross signing key.
 #[doc = include_str!(concat!(env!("CARGO_MANIFEST_DIR"), "/src/doc/string_enum.md"))]
-#[derive(Clone, StringEnum)]
+#[derive(Clone, StringEnum, PartialEq, Eq)]
```

Comment:
Third-party derives after standard library ones?

```suggestion
#[derive(Clone, PartialEq, Eq, StringEnum)]
```